yarn.lock

```diff
@@ -4335,11 +4335,6 @@ highlightjs-badge@0.1.9:
   resolved "https://registry.yarnpkg.com/highlightjs-badge/-/highlightjs-badge-0.1.9.tgz#39cec0ff9d55d4ca02ea04ab31e376331f66d9b1"
   integrity sha512-uWA5OvsV3zTnzlXWv12AymBwWwH7VjzndiX/ts2O4NXnM6ezpbkjMKfB6WtYF84syNltoOZgMrluPvwI4utrUw==
 
-highlightjs-line-numbers.js@^2.8.0:
-  version "2.8.0"
-  resolved "https://registry.yarnpkg.com/highlightjs-line-numbers.js/-/highlightjs-line-numbers.js-2.8.0.tgz#479ea8cff0c31fadc1578a66fa03e38b801f9ca6"
-  integrity sha512-TEf1gw0c8mb8nan0QwliqS7obT4cpUd9hzsGzsZLweteNnWea/VIqy5/aQqsa5wnz9lnvmtAkS1ZtDTjB/goYQ==
-
```

Comment:
Why is this in your diff?

Reply:
It popped up after i yarn installed to use the newest yarn dev. I could do a git checkout and revert it?